lms/static/js/verify_student/views/incourse_reverify_view.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * View for in-course reverification.
+ *
+ * This view is responsible for rendering the page
+ * template, including any subviews (for photo capture).
+ */
+ var edx = edx || {};
+
+(function( $, _, Backbone, gettext ) {
+    'use strict';
+
+    edx.verify_student = edx.verify_student || {};
+
+    edx.verify_student.InCourseReverifyView = Backbone.View.extend({
+        el: '#incourse-reverify-container',
+        templateId: '#incourse_reverify-tpl',
+        submitButtonId: '#submit',
+
+        initialize: function( obj ) {
+            this.errorModel = obj.errorModel || null;
+            this.courseKey = obj.courseKey || null;
+            this.checkpointName = obj.checkpointName || null;
+
+            this.model = new edx.verify_student.ReverificationModel({
+                courseKey: this.courseKey,
+                checkpointName: this.checkpointName
+            });
+
+            this.listenTo( this.model, 'sync', _.bind( this.handleSubmitPhotoSuccess, this ));
+            this.listenTo( this.model, 'error', _.bind( this.handleSubmissionError, this ));
```

Comment:
I would add this.render() here so whenever a new InCourseReverifyView is created it renders straight away (unless there is a use case where that shouldn't occur).